packages/hoppscotch-backend/package.json

```diff
@@ -30,86 +30,84 @@
+    "@nestjs/config": "4.0.2",
+    "@nestjs/core": "11.1.1",
+    "@nestjs/graphql": "13.1.0",
+    "@nestjs/jwt": "11.0.0",
+    "@nestjs/passport": "11.0.0",
+    "@nestjs/platform-express": "11.1.1",
+    "@nestjs/schedule": "6.0.0",
+    "@nestjs/swagger": "11.2.0",
+    "@nestjs/terminus": "11.0.0",
+    "@nestjs/throttler": "6.4.0",
+    "@prisma/client": "6.8.2",
+    "argon2": "0.43.0",
+    "bcrypt": "6.0.0",
     "class-transformer": "0.5.1",
-    "class-validator": "0.14.1",
-    "cookie": "1.0.0",
+    "class-validator": "0.14.2",
+    "cookie": "1.0.2",
     "cookie-parser": "1.4.7",
-    "cron": "3.1.7",
-    "express": "4.21.1",
+    "cron": "4.3.0",
+    "express": "5.1.0",
     "express-session": "1.18.1",
-    "fp-ts": "2.16.9",
-    "graphql": "16.9.0",
-    "graphql-query-complexity": "1.0.0",
-    "graphql-redis-subscriptions": "2.6.1",
-    "graphql-subscriptions": "2.0.0",
+    "fp-ts": "2.16.10",
+    "graphql": "16.11.0",
+    "graphql-query-complexity": "1.1.0",
+    "graphql-redis-subscriptions": "2.7.0",
+    "graphql-subscriptions": "3.0.0",
     "handlebars": "4.7.8",
-    "io-ts": "2.2.21",
-    "luxon": "3.5.0",
-    "nodemailer": "6.9.15",
+    "io-ts": "2.2.22",
+    "luxon": "3.6.1",
+    "nodemailer": "7.0.3",
     "passport": "0.7.0",
     "passport-github2": "0.1.12",
     "passport-google-oauth20": "2.0.0",
     "passport-jwt": "4.0.1",
     "passport-local": "1.0.0",
     "passport-microsoft": "2.1.0",
-    "posthog-node": "4.2.0",
-    "prisma": "5.20.0",
+    "posthog-node": "4.17.1",
+    "prisma": "6.8.2",
     "reflect-metadata": "0.2.2",
     "rimraf": "6.0.1",
-    "rxjs": "7.8.1"
+    "rxjs": "7.8.2"
   },
   "devDependencies": {
-    "@nestjs/cli": "10.4.5",
-    "@nestjs/schematics": "10.1.4",
-    "@nestjs/testing": "10.4.4",
+    "@nestjs/cli": "11.0.7",
+    "@nestjs/schematics": "11.0.5",
+    "@nestjs/testing": "11.1.1",
     "@relmify/jest-fp-ts": "2.1.1",
     "@types/bcrypt": "5.0.2",
-    "@types/cookie": "0.6.0",
-    "@types/cookie-parser": "1.4.7",
-    "@types/express": "5.0.0",
-    "@types/jest": "29.5.13",
-    "@types/luxon": "3.4.2",
-    "@types/node": "22.7.5",
-    "@types/nodemailer": "6.4.16",
+    "@types/cookie-parser": "1.4.8",
+    "@types/express": "5.0.2",
+    "@types/jest": "29.5.14",
+    "@types/luxon": "3.6.2",
+    "@types/node": "22.15.19",
+    "@types/nodemailer": "6.4.17",
     "@types/passport-github2": "1.2.9",
     "@types/passport-google-oauth20": "2.0.16",
     "@types/passport-jwt": "4.0.1",
     "@types/passport-microsoft": "1.0.3",
-    "@types/supertest": "6.0.2",
-    "@typescript-eslint/eslint-plugin": "8.8.1",
-    "@typescript-eslint/parser": "8.8.1",
+    "@types/supertest": "6.0.3",
+    "@typescript-eslint/eslint-plugin": "8.32.1",
+    "@typescript-eslint/parser": "8.32.1",
     "cross-env": "7.0.3",
-    "eslint": "8.57.0",
-    "eslint-config-prettier": "9.1.0",
-    "eslint-plugin-prettier": "5.2.1",
+    "eslint": "9.27.0",
```

Comment:
ESLint `v9` has breaking changes that need to be addressed.

<img width="793" alt="image" src="https://github.com/user-attachments/assets/774fdb58-d4b2-4257-a3a2-efa433441afa" />

Reply:
Migrated to v9